report/chap/parity.tex

```diff
@@ -1,95 +0,0 @@
-
-Apart from Parity, the following chapters will also introduce Mean and Discounted Payoff Games. Although seemingly unrelated, each game can be reduced to the successive one. This is used in \cite{parityup} to show that the decision problems for all of them are in $\UP \cap \coUP$ (here $\UP$ is the class of decision problems that have a unique polynomial size certificate, hence $\P \subseteq \UP \subseteq \NP$). Whether any of these games (i.e. their respective decision problems) has a polynomial time solution or not is again an open question.
-
-\section{Definition}
-%TODO: Make this consitent with notation in Chapter 1
-%Idea: Call it G instead of \G like done in Mean Payoff
-Let $G=(V_0,V_1,E)$ be a graph and let (for some $M \in \Nat$)
-\[
-    p : V \mapsto \set{0,1,2 \ldots M}
-\]
-be an assignment of integer priorities to each vertex.
-
-Parity game $\G_p=(G,Win_p)$ is a Win-Lose game (section \ref{sec:winlose}) with
-\begin{align}
-    Win_p &= \set{ \left(v_0 v_1 \ldots\right) \in \PI \mid \left(\limsup_{i \geq 0} p(v_i)\right)  \text{ is Odd }} \label{cond:parity}
-\end{align}
-
-Let the largest priority seen infinitely often, on a path $\pi$, be denoted by \mip\ of $\pi$. If \mip\ of $\pi$ is odd then \Player{1} wins $\pi$; if the \mip\ is even then \Player{0} wins. Since $V$ is finite, there's a vertex $v$ which is visited infinitely often by $\pi$, and has the same priority as the \mip.
-
-Notice that in contrast with finite games (section \ref{sec:finitegames}), the winning condition \eqref{cond:parity} only depends on long run behavior. This property is also called prefix independence -- for $w \in \PF$ and $\alpha \in \PI$ if $w \cdot \alpha \in \PI$ then $\alpha \in Win_p \iff w \cdot \alpha \in Win_p$. However, due to path constraints in the graph G, the prefix might still be important as it can enable or disable certain long term behaviors.
-
-
-\section{Positional Determinacy}
-From every vertex $v \in V$ the parity game $(\G_p,v)$ is determined (section \ref{subsec:determinacy}) -- one can invoke the general determinacy theorem by Martin \cite{martin_borel_1975}  to show this. However a stronger result was proved by Emerson \cite{emerson_automata_1985} --\\
-
-
-\begin{theorem}
-    \label{thm:posParity}
-    For any parity game $\G_p=(G,Win_p)$, there are positional strategies $(\sigma^*,\tau^*)$ and a partition of $V = W_0 \sqcup W_1$, so that $\sigma^*$ is a winning strategy for \Player{0} from each $v \in W_0$ and $\tau^*$ is a winning strategy for \Player{1} from each $v \in W_1$.
-\end{theorem}
-Hence, not only is the game $(\G_p,v)$ determined, the winner has a positional winning strategy independent of $v$ (in the player's winning region).
-
-Given $v \in V$ and positional strategies $(\sigma,\tau)$, since $\pi^{v}_{\sigma\tau}$ is ultimately periodic we have 
```

Comment:
You could perhaps add 1-2 lines to say why the play determined by positional strategies \sigma and \tau is ultimately periodic.